conf/azure.yaml.template

```diff
@@ -4,8 +4,10 @@ AZURE:
         SECRET_ID:
         TENANT_ID:
         SUBSCRIPTION_ID:
-        RESOURCE_GROUP:
-        REGION:
+        # Multiple resource groups can be added like ["foo", "bar"] or ["all"] for all RGs
+        RESOURCE_GROUPS: ["foo", "bar"]
```

Comment:
IMO the template should contain empty array []. Having an example in the comment is a great idea though!

Reply:
With this logic shouldn't i also change [deleteble prefix](https://github.com/laDok8/cloudwash/blob/feature.azureMatrix/settings.yaml.template#L52) from `'test'` to `''`?

Reply:
Yup! Feel free to do that!